### ISSUE TITLE
TypeError: fetch_books() got an unexpected keyword argument 'commit'

### ISSUE THREAD

**Issue by simonw** [rixx/goodreads-to-sqlite#1]
```/tmp $ goodreads-to-sqlite books books.db
Traceback (most recent call last):
  File "/Users/simonw/Library/Python/3.7/bin/goodreads-to-sqlite", line 10, in <module>
    sys.exit(cli())
  File "/usrTypeError: fetch_books() got an unexpected keyword argument 'commit'/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/simonw/Library/Python/3.7/lib/python/site-packages/goodreads_to_sqlite/cli.py", line 109, in books
    utils.fetch_books(db, user_id, token, commit=True)

/tmp $ 
```

**Comment by simonw** [comment]
This is from a fresh `pip install goodreads-to-sqlite` just now

```
/tmp $ goodreads-to-sqlite --version
goodreads-to-sqlite, version 0.1
```

**Comment by simonw** [comment]
Looks like it might be fixed:

https://github.com/rixx/goodreads-to-sqlite/commit/a2feb3193c97f949fe205d5f9bb36537174ae2e0#diff-1bebf16cf8e2223fd28a0ff5b9b7fdbcR79-R80

**Comment by simonw** [comment]
Needs a release to PyPI to resolve this though.

**Comment by rixx** [comment]
Will release a second version today, I'm just seeing if I can get webscraping in there, too.

**Comment by rixx** [comment]
Bug fix and web scraping is available in v0.2!

**Comment by simonw** [comment]
Confirmed, this works for me now and it's fantastic.
